src/kademlia/dht_storage.cpp

```diff
@@ -106,7 +105,7 @@ namespace
 
 	void touch_item(dht_immutable_item& f, address const& addr)
 	{
-		f.last_seen = aux::time_now();
+		f.last_seen = total_seconds(aux::time_now().time_since_epoch());
```

Comment:
unfortunately this won't work either. I forget, the idea is to persist this timestamp to disk, right? The epoch of the steady clock is likely to be something like: time of last boot. You need a real-time clock, or wall clock, here. I would imagine you'd want to use ``std::chrono::system_clock`` and use ``to_time_t``.

(i.e. ``libtorrent::clock_type`` is ``steady_clock``, which just guarantees that it is monotonically increasing, no guarantee about having a fixed epoch)